src/plugins/tracking/activity.js

```diff
@@ -17,9 +17,18 @@ export const getLogger: string => (Object, Object) => mixed = (
 	agent: string
 ) => (request: Object, trackInfo: Object) => {
 	const requestHeaders = request.headers;
-	const now: Date = new Date();
+	
+	// Takes in desired time to convert and applies offset
+	const offset: String = new Date().getTimezoneOffset() * 60000; // gets detected timezone + converts to milliseconds
```

Comment:
Looks like our PR builder isn't catching Flow errors - this and `NY_OFFSET` are not strings. You also shouldn't have to declare types on variables with types that can be easily inferred (e.g. this Number and the Date vars).